cronograma/r-dev-day/index.qmd

```diff
@@ -70,11 +70,11 @@ Algunas cosas que puedes hacer para ayudar a prepararte si tienes tiempo:
 
 La sección de [Otros recursos](#otros-recursos) contiene enlaces para aprender más sobre cómo contribuir. Revisar estos enlaces con antelación es totalmente opcional (intentaremos asignarte las tareas en función de tus conocimientos y experiencia actuales), pero te ayudarán a ponerte manos a la obra en el evento.
 
-Si tienes alguna pregunta, no duded en ponerte en contacto con uno de los miembros del comité organizador a través del Slack de LatinR:
+Si tienes alguna pregunta, no dudes en ponerte en contacto con uno de los miembros del comité organizador a través del Slack de LatinR:
 
 | Hablantes de español        | Hablantes de portugués | Hablantes de inglés |
 |-----------------------------|------------------------|---------------------|
-| Pao Corrales                | Beatriz Milz           | gwynn gebeyhu       |
+| Pao Corrales                | Beatriz Milz           | Gwynn Gebeyhu       |
```

Comment:
From what I remember, gwynn's always spell on lower case: https://www.nzgwynn.com/

@PatriLoto @hturner

Reply:
you're right!